src/reference/v8.json

```diff
@@ -34,6 +34,11 @@
         40.7736
       ]
     },
+    "centerAltitude": {
```

Comment:
Are we expecting other altitudes? Why not simply "altitude"?

Reply:
I am not expecting other altitudes in the stylesheet, but centerAltitude makes it clear that 1) it is related to the existing `center` field, and 2) it is not the camera altitude.

Reply:
Sorry if this is not a camera altitude, then what is the altitude of?

Reply:
The center of the map, what/where the camera is looking at.

Reply:
OK I get it now... Sorry I am a bit slow.

I honestly don't understand the use case of this property (or `bearing`, `pitch`, `roll` or even `center` for that matter). I'm with @wipfli that this should be done with an API.

> I know `center` and `zoom` are already part of the style spec so we will not remove them, but in my opinion they should never have been added. I think the style sheet should encode how the map looks like, and the MapLibe GL JS / Native API should take care of what you do with the map, i.e., where to look at, zoom level, field of view etc...

I'm can't think of a reason why you'd want to put these in a style.

Fun fact: `center` is broken on MapLibre Native because as part of map initialization the map is shifted a bit so it detects 'map interaction' and doesn't set the center... Nobody has complained about it in 5 years.

Reply:
While this might be hard to do, try to put yourself in the shoes of a map designer, when you don't know how to write code at all, and even a json file is not a straight forward concept.
You work on maputnik for example, style the map set where the center, zoom, pitch etc is and export the style to be consumed by a different team, or even export the html page that maputnik exports. 
When these are defined in the style all you need to do is edit a json, no code, less complexity, and as a designer, you have fun control of how it would look. 
As opposed to what you might be familiar with, in big corporations, these split between those who wrote code and those that doesn't is very common, and the time of those that write code is usually very expensive, so as an organization, you try to facilitate as much as possible without needing someone to code...

Reply:
Using that same argument we can add animations and user interactions, UI controls and a whole suite of other features that have little to do with how a map looks to the spec.

> You work on maputnik for example

Right now you have to input numbers in the Center/Zoom/Bearing/Pitch in Maputnik. There's no way to preview the values you entered or copy the current settings from the map. So I think (at least now) the workflow you describe is hypothetical.

Reply:
Doesn't this property conflict with `zoom` and probably `pitch` as well? Won't this over-specifying the initial camera position?

Anyway, I'm pretty sure this property is not really relevant for MapLibre Native. I'll just trust @HarelM's and @ibesora's judgement on this.

Reply:
> Doesn't this property conflict with zoom and probably pitch as well? Won't this over-specifying the initial camera position?

It’s only used if no other means of setting the camera have been used and it enables the “orbiting around a point not in the terrain” use case.

I’m with @HarelM about this making sense in the style. To me, where the camera is looking is an integral part of how a map looks like, similar to center.